examples/donation-balancesOf/index.mjs

```diff
@@ -0,0 +1,74 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+import launchToken from '@reach-sh/stdlib/launchToken.mjs';
+
+const stdlib = loadStdlib(process.env);
+
+(async () => {
+  // Make accounts
+  const startingBalance = stdlib.parseCurrency(100);
+  const [ accAlice, accBob, accCreator ] =
+    await stdlib.newTestAccounts(3, startingBalance);
+
+  // Create tokens
+  const chicken = await launchToken(stdlib, accCreator, "chicken", "CKN");
+  const egg = await launchToken(stdlib, accCreator, "egg", "EGG");
+
+  // On ALGO, Alice & Bob's accounts have to accept CKN and EGG before they can receive it
+  // Note: this costs algo because it's recorded on the blockchain
+  if (stdlib.connector == 'ALGO') {
```

Comment:
You don't need to protect this code, just always do it and on connectors that don't need it, nothing happens.